### ISSUE TITLE
Meta informations are pronounced in the language set in voice preferences, not in the page

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5161]
**Reported by jmuheim on 2015-06-18 08:22**
I noticed that NVDA always pronounces meta informations (e.g. the infos about a text input) in the language that's set in the voice preferences, and not in the language of the document.

This makes sense in many cases, as I like to hear those informations in my default language and not e.g. in arabic when browsing an arabic page.

But there are some edge cases, where this isn't correct in my opinion. For example, when browsing a table, then the associated header cells (whose contents are in the page's language) are also pronounced in NVDA's set language, and not in the page's language. It seems that NVDA doesn't distinguish between meta data that comes from technical stuff and meta data that is defined by the webpage's content.

I have set up an example here:

http://codepen.io/jmuheim/pen/doVPqg

Just set a language other than english in your voice preferences, and then you'll notice that "Vorname"
 and "Nachname" are not pronounced in german (although the page's language is set to german, and other text is pronounced well), but in the set language.

**Comment by ehollig** [comment]
I am not sure I understand the original issue. Also, the link provided in https://github.com/nvaccess/nvda/issues/5161#issuecomment-155340424 no longer works. CC @jmuheim

**Comment by jmuheim** [comment]
@gurix: This is exactly what I talked to you about during the last WhatsApp voice memo. Maybe you can try to reproduce it and (if necessary) give more explanations to @ehollig?

**Comment by Adriani90** [comment]
@jmuheim, @gurix, @ehollig I have tested as following:
1. Open NVDA and in synth settings, enable automatic switch language if supported and recognize dialects as well.
2. In document formating settings, I have enabled "table" and "column and row headings"
3. On the website in the issue description, I have switched to different synthesizers (one core, eSpeak and eloquence)
4. I choose different voice languages (tested with finish, romanian, greek, spanish, brasilian)

Results: the automatic language detection worked properly. In the table the headers were read in german. under the table, where the english part began, the voice switched automatically to english.
So, I cannot reproduce this issue.
Are you still able to reproduce it? If yes, in which browser?
